zerver/views/registration.py

```diff
@@ -198,7 +199,7 @@ def accounts_register(request):
             do_set_user_display_setting(user_profile, 'timezone', timezone)
```

Comment:
I think technically we should do a `do_change_is_admin` here in this code path; but it's unlikely to come up and should be mostly harmless until then.  I've been planning to rewrite this section, so maybe just add a comment:
"# TODO: When we clean up this code path, make it respect is_realm_admin." ?